airflow/utils/cli.py

```diff
@@ -140,10 +141,17 @@ def _build_metrics(func_name, namespace):
     :return: dict with metrics
     """
     sub_commands_to_check = {"users", "connections"}
+    sub_commands_to_check_based_upon_key = {"variables"}
     sensitive_fields = {"-p", "--password", "--conn-password"}
     full_command = list(sys.argv)
     sub_command = full_command[1] if len(full_command) > 1 else None
-    if sub_command in sub_commands_to_check:
+    # For cases when value in variables have sensitive value
+    if sub_command in sub_commands_to_check_based_upon_key:
+        key = full_command[-2] if len(full_command) > 3 else None
```

Comment:
This is the full command right:
```
"full_command": "['/usr/local/bin/airflow', 'variables', 'set', '--description', 'my-secret', 'client_secret_2149', '********']"}
```

So we need to get -1 of it since -1 would always be "value".

Reply:
`airflow variables import` has a 4th parameter, so does many other commands, they have lot of extra options. So better to check that its a `airflow variables set` instead.

Reply:
This is for keys,
We are checking the keys for the value to be masked. If the key contains some sensitive word, mask the value.

Reply:
Wondering it using `mask_secret` would instead do a better job here, it would hide if by adding it as a secret mask and every further occurence will be taken care of.
cc @potiuk

Reply:
I think this is under wrapper and used before action logging, we need to change the value at this moment only. They  have implemented same design for -p (password) instead of using the mask_secret and saving it's pattern, they changed here itself, since this itself will be logged.

Reply:
Yeah. Also it's good for the future, when we will separate API package (after 3.0 likely) -this is way more "standalone" and simple.